usb_winusb.go

```diff
@@ -58,19 +59,20 @@ func winusbError(err error) error {
 	return nil
 }
 
-func usbopen(cid string) (err error) {
+func usbopen(cid string, timeout time.Duration, serial string) (err error) {
 	if device.ctx != nil {
 		log.WithField("Correlation-ID", cid).Debug("usb context already open")
 		return nil
 	}
 
+	ms := C.ulong(timeout.Milliseconds())
```

Comment:
This doesn't seem to happen (tested a bit). What I see is:
--timeout 1
then timeout = time.Duration(1) * time.Millisecond == time.Duration(1000000) // counts in nanoseconds
then timeout.Milliseconds() == 1
because it's returning the milliseconds in timeout

Reply:
Ah, you're absolutely right. My bad.